converters/fromZigbee.js

```diff
@@ -430,6 +430,53 @@ const converters = {
             }
         },
     },
+    level_config: {
+        cluster: 'genLevelCtrl',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const result = {'level_config': {}};
+
+            // onOffTransitionTime - range 0x0000 to 0xffff - optional
+            if (msg.data.hasOwnProperty('onOffTransitionTime') && (msg.data['onOffTransitionTime'] !== undefined)) {
+                result.level_config.onOffTransitionTime = msg.data['onOffTransitionTime'];
+            }
+
+            // onTransitionTime - range 0x0000 to 0xffff - optional
+            //                    0xffff = use onOffTransitionTime
+            if (msg.data.hasOwnProperty('onTransitionTime') && (msg.data['onTransitionTime'] !== undefined)) {
+                result.level_config.onTransitionTime = msg.data['onTransitionTime'];
+                if (result.level_config.onTransitionTime == 65535) {
```

Comment:
Can you make all published properties snake  case (e.g. `on_transition_time`)?

Reply:
Sure, i think I saw a function to translate snake to camel somewhere in zhc recently, let me poke around.